src/d3a/models/strategy/external_strategies/__init__.py

```diff
@@ -166,17 +193,25 @@ def _reset_event_tick_counter(self):
         self._last_dispatched_tick = 0
 
     def _dispatch_event_tick_to_external_agent(self):
+        if not self.connected and not self.is_aggregator_controlled:
```

Comment:
This will disable the publishing of the tick event to "normal" external clients, right?

Reply:
No actually, it will allow it. This clause early-returns if neither an external client nor an aggregator is connected, therefore if either of the 2 is connected, the tick event will be published.